### ISSUE TITLE
Screenly Queue

### ISSUE THREAD

**Issue by vpetersson** [Screenly/Playground#122]
The idea behind Screenly Queue is to develop a queue system for retail environments that is open source and cost effective. 

**tl;dr:** The user walks in to the store and gets a number. The teller presses button in the user interface (teller interface)  and the number shows up on the screen (dashboard interface).

# Design

There are four sides to the app:

 * Admin: Store manager
 * Teller: Day-to-day user
 * Dashboard: The signage side (Edge App)
 * Customer: A way to get a number in the line

## Admin

The admin interface is simply the Django admin interface. You will be able to do the following:

 * Add/remove users
 * Add/remove counters
 * Reset/manage the ticket queue(s)

In the prototype we will only have one queue, but it would be interesting down the road to have different queues. 

## Teller

The teller interface is a Django page that allows you to do the following:

 * Select a counter (probably one page counter)
 * Skip to the next in the counter

## Dashboard

This is the interface that will be shows on a big monitor. It's just an Edge App that pulls from the main server (to ensure rapid updates). It will show "Now serving X at counter Y". When the number is increased, the display will jump forward with very little latency.

## Customer

This is where the customer gets the number from (if the client isn't using a physical printer). It's simply a page that will give you a session cookie and when you press "Request number" it return a number.  This is normally displayed used on a mobile phone.

**Comment by salmanfarisvp** [comment]
Moved to the Egde App List.
